### PR TITLE
Feat/visitor

### PR DESCRIPTION
**Pull request by Mnfriedl**
Modelo y rutas para crear/modificar/eliminar y lista de visitantes.

Las rutas están en Postman.

### CODE REVIEW

File: entrance_control/urls.py [hunk 1]
```diff
@@ -23,5 +23,6 @@
     path('api/', include('api.urls')),
     path('users/', include('users.urls')),
     path('plates/', include('plates.urls')),
-    path('', include('providers.urls'), name="providers"),
+    path('', include('providers.urls'), name='providers'),
```
Comment (bimartinez) [inline entrance_control/urls.py:26]: Why here we don't have this?
```suggestion
    path('providers/', include('providers.urls'), name='providers'),
```

File: visitors/models.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from users.models import Resident
+
+
+# Create your models here.
+class Visitor(models.Model):
+    name = models.CharField(max_length=100)
+    lastname = models.CharField(max_length=100)
+    rut = models.CharField(max_length=10)  # 01.234.567-9
```
Comment (bimartinez) [inline visitors/models.py:9]: I think the format is without separating dots
```suggestion
    rut = models.CharField(max_length=10)  # 01234567-9
```

File: visitors/serializers.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+class VisitorSerializer(serializers.ModelSerializer):
+
+    permission_classes = [
+            permissions.IsAuthenticated,
+        ]
+
+    class Meta:
+        model = Visitor
+        fields = [
```
Comment (bimartinez) [inline visitors/serializers.py:15]: We should use single quotes pls

File: visitors/serializers.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+class ListVisitorSerializer(serializers.ModelSerializer):
+    permission_classes = [
+            permissions.IsAuthenticated,
+        ]
+
+    resident = ResidentSerializer()
+
+    class Meta:
+        model = Visitor
+        fields = [
```
Comment (bimartinez) [inline visitors/serializers.py:41]: Same here, single quotes pls

File: visitors/urls.py [hunk 5]
```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+import visitors.api as api
+
+urlpatterns = [
+    path("visitors/", api.VisitorsList.as_view()),
```
Comment (bimartinez) [inline visitors/urls.py:5]: Here you have to remove the `visitors/` prefix and add it on the main `urls.py` file, also use single quotes

File: entrance_control/urls.py [hunk 6]
```diff
@@ -23,5 +23,6 @@
...
+    path('', include('visitors.urls'), name='visitors'),
```
Comment (bimartinez) [inline entrance_control/urls.py:27]: Same here
```suggestion
    path('visitors/', include('visitors.urls'), name='visitors'),
```

**Review by bimartinez — CHANGES_REQUESTED** [review]

**Review by fgbruna — APPROVED** [review]

File: visitors/models.py [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+# Create your models here.
+class Visitor(models.Model):
+    name = models.CharField(max_length=100)
+    lastname = models.CharField(max_length=100)
+    rut = models.CharField(max_length=10)  # 01234567-9
+    resident = models.ForeignKey(Resident, null=True, on_delete=models.SET_NULL, editable=False)
+
+    def __str__(self):
+        return f"{self.name} {self.lastname}"
```
Comment (bimartinez) [inline visitors/models.py:13]: Use single quotes please

**Review by bimartinez — CHANGES_REQUESTED** [review]

**Review by bimartinez — APPROVED** [review]

**Pull request merged**